src/file-download/file-list-card/file-list-card.styles.tsx

```diff
@@ -0,0 +1,216 @@
+    width: 100%;
+    border: none;
+
+    :not(:last-child) {
+        margin-bottom: 1rem;
+    }
+`;
+
+export const DragHandleIcon = styled(
+    DSDragHandleIcon
+)<DragHandleIconStyleProps>`
+    // Temp icon
+    margin-right: 1rem;
+    height: 1.5rem;
+    width: 1.5rem;
+
+    ${(props) => {
+        if (props.$disabled) {
+            return css`
+                color: ${Color.Neutral[4]};
+            `;
+        }
+    }}
+`;
+
+export const Box = styled.div`
+    background: ${Color.Accent.Light[6]};
+    border: 1px solid ${Color.Neutral[5]};
+    border-radius: 4px;
+    padding: 1rem 2rem;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    cursor: pointer;
+    ${MediaQuery.MaxWidth.mobileL} {
+        padding: 1rem;
+    }
+    :hover {
+        background: ${Color.Accent.Light[5]};
+    }
+`;
+
+export const ContentSection = styled.div<ContentSectionStyleProps>`
+    display: flex;
+    flex: 1;
+    align-items: center;
+
+    ${MediaQuery.MaxWidth.mobileL} {
+        flex-direction: column;
+        width: 100%;
+        align-items: flex-start;
+    }
+
+    ${(props) => {
+        if (props.$hasThumbnail) {
+            return css`
+                ${MediaQuery.MaxWidth.mobileL} {
+                    flex-direction: row;
+                    align-items: center;
+                }
+            `;
+        }
+    }}
+`;
+
+export const NameSection = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    width: 100%;
+    max-width: 22.688rem;
+`;
+
+export const ExtendedNameSection = styled.div`
+    display: flex;
+    flex: 1;
+    align-items: center;
+
+    ${MediaQuery.MaxWidth.mobileL} {
+        flex-direction: column;
+        align-items: flex-start;
+        width: 100%;
+    }
+`;
+
+export const FileSizeSection = styled.div<FileSizeSectionStyleProps>`
+    display: flex;
+    width: 5rem;
+    margin-left: 0.5rem;
+    justify-content: flex-end;
+
+    ${MediaQuery.MaxWidth.mobileL} {
+        ${(props) => {
+            if (props.$hideInMobile) {
+                return css`
+                    display: none;
+                    visibility: hidden;
+                `;
+            } else {
+                return css`
+                    width: 100%;
+                    margin-left: 0;
+                    margin-top: 0.5rem;
+                    justify-content: flex-start;
+                `;
+            }
+        }}
+    }
+`;
+
+export const ItemText = styled(Text.BodySmall)``;
+export const ItemDescriptionText = styled(ItemText)`
+    margin-top: 0.25rem;
+`;
+
+export const BaseErrorMessage = styled(Text.XSmall)`
+    font-size: 0.875rem !important;
+    color: ${Color.Validation.Red.Text};
+`;
+
+export const DesktopErrorMessage = styled(BaseErrorMessage)`
+    margin-top: 0.25rem;
+    ${MediaQuery.MaxWidth.mobileL} {
+        display: none;
+        visibility: hidden;
+    }
+`;
+
+export const MobileErrorMessage = styled(BaseErrorMessage)`
```

Comment:
this seems to be unused, but it should be display on mobile too

Reply:
is this still a WIP? as in the latest commit it was imported but commented out

Reply:
it's still a WIP, just got some update from UI/UX so i'll update the code latter